bin/deploy_k8s

```diff
@@ -109,8 +109,10 @@ upload_releases() {
 }
 
 upload_stemcell() {
-  local stemcell_url="$(get_setting director.yml /stemcell_url)"
-  BOSH_CLIENT=bosh_admin BOSH_CLIENT_SECRET="$(get_bosh_secret)" bosh-cli -e "${BOSH_NAME}" upload-stemcell ${stemcell_url}
+  if [ "${release_source}" != "skip" ]; then
```

Comment:
${release_source} is local to the `main` function and shouldn't be visible here.
Please add a test to make sure it works as planned.